### PR TITLE
swift 4.0

### PR DESCRIPTION
**Pull request by krezzoid**
swift 4.0
fix sample project
bump spec version

### CODE REVIEW

**Comment by ricardopereira** [comment]
@krezzoid Thanks a lot!

The lib wasn't working on Xcode 9? I think I tried it and it was working well.

**Comment by ricardopereira** [comment]
Reference: #52

File: WSTagsField.podspec [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  s.version          = "2.2.0"
   s.summary          = "An iOS text field that represents different Tags"
   s.homepage         = "https://github.com/whitesmith/WSTagsField"
   s.license          = 'MIT'
   s.author           = { "Ricardo Pereira" => "m@ricardopereira.eu" }
   s.source           = { :git => "https://github.com/whitesmith/WSTagsField.git", :tag => s.version.to_s }
   s.social_media_url = 'https://twitter.com/whitesmithco'
 
-  s.platform     = :ios, '8.0'
+  s.platform     = :ios, '9.0'
```
Comment (ricardopereira) [inline WSTagsField.podspec]: Why did you removed 8.0 compatibility?

File: WSTagsField.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -296,11 +302,12 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (ricardopereira) [inline WSTagsField.xcodeproj/project.pbxproj]: Ok, now I understand 😄

**Comment by ricardopereira** [comment]
I need to update Bitrise worker to use Xcode 9.

**Comment by loopingGIT** [comment]
Hey is there a way of using WSTagsfield on Swift 4? I was using it on swift 3, and now is not longer working.
Thanks.

**Comment by krezzoid** [comment]
@loopingGIT just use sources from matter branch, this version perfectly works with swift4 and latest version of Xcode

**Comment by ricardopereira** [comment]
@loopingGIT This has been merged, so it works using the `master` branch. A release with Swift 4 is missing because there's a critical problem that is still open (#65)

**Comment by krezzoid** [comment]
@ricardopereira could I help you with #65? I could solve it if have a time, but I can't understand the issue - not clear description :(

**Comment by ricardopereira** [comment]
@krezzoid WSTagsField has a wrong layout when it's used in a `UITableView`.

**Comment by krezzoid** [comment]
@ricardopereira do you have sample code?

**Comment by ricardopereira** [comment]
@krezzoid There's an example using a table view. I think i fixed it. Thanks. The PR with the fix is #69. I need to do more tests. Feel free to have a look.

**Pull request merged**